resources/default_site.rb

```diff
@@ -17,16 +17,16 @@
          description: 'Cookbook to source the template file from'
 
 property :server_admin, String,
-         default: 'admin@server',
+         default: 'root@localhost',
          description: 'Default site contact name'
 
 property :log_level, String,
          default: 'warn',
          description: 'log level for apache2'
 
 action :enable do
-  template 'default-site.conf' do
-    path "#{apache_dir}/sites-available/default-site.conf"
+  template "#{new_resource.default_site_name}.conf" do
```

Comment:
Does this need explanation that if you change the default site name you need to ensure you have a template?

Reply:
Yeah we probably need loads of things noting in this PR. 

It also needs to work like the specified platform